src/test/unit_tests/jest_tests/test_nc_nsfs_account_cli.test.js

```diff
@@ -103,6 +103,28 @@ describe('manage nsfs cli account flow', () => {
             assert_account(account_symlink, account_options);
         });
 
+        it('should fail - cli create account invalid access_key(leading space in access_key)', async () => {
+            const { type, name, uid, gid, secret_key } = defaults;
+            const account_options = { config_root, name, uid, gid, secret_key};
+            const action = ACTIONS.ADD;
+            const command = create_command(type, action, account_options)
+            const flag = 'access_key';
+            const value = ' 12345678912345678ABC'; // leading space for access_key
+            const res = await exec_manage_cli_add_leading_space_option(command, flag, value);
+            expect(JSON.parse(res.stdout).error.message).toBe(ManageCLIError.InvalidArgument.message);
+        });
+
+        it('should fail - cli create account invalid secret_key(leading space in secret_key)', async () => {
+            const { type, name, uid, gid, access_key } = defaults;
+            const account_options = { config_root, name, uid, gid, access_key};
+            const action = ACTIONS.ADD;
+            const command = create_command(type, action, account_options)
+            const flag = 'secret_key';
+            const value = ' a234567891234567891212345678912345678912'; // leading space for secret_key
```

Comment:
I think you need to have EXAMPLE suffix for secret_key property (to avoid creating alerts on the published commit).

Reply:
Could you explain it more?

Reply:
I'll send you here an example:
https://github.com/noobaa/noobaa-core/blob/f32827bba83a2ecfd403fbfb83cd81e50fbe0c2c/src/test/unit_tests/jest_tests/test_nc_nsfs_account_cli.test.js#L40

I'll send you in Slack the title of the mail you might have got so you can handle it.

Reply:
I have updated the logic and we need not to assign `secret_key` value like this any more. Thanks for raising this point.